core/lib/zksync_core/src/lib.rs

```diff
@@ -1294,11 +1330,17 @@ async fn run_http_api(
     with_debug_namespace: bool,
     storage_caches: PostgresStorageCaches,
     mempool_cache: MempoolCache,
+    tx_sink_config: Option<TxSinkConfig>,
 ) -> anyhow::Result<()> {
+    let config = TxSenderBuilderConfigs {
+        tx_sender_config: tx_sender_config.clone(),
```

Comment:
Why `clone`?

Reply:
To construct the `TxSenderBuilderConfigs` takes the ownership of the value in `tx_sender_config` (it's a reference). So the compiler requires `clone` the value. 

If there any better way to construct the struct and assign it to `build_tx_sender`? I made this change  because of the linting error (too many arguments in build_tx_sender)